.gitmodules

```diff
@@ -1,15 +1,6 @@
 [submodule "third_party/stxxl"]
 	path = third_party/stxxl
 	url = https://github.com/ad-freiburg/stxxl
-[submodule "third_party/googletest"]
-	path = third_party/googletest
-	url = https://github.com/google/googletest.git
-[submodule "third_party/re2"]
-	path = third_party/re2
-	url = https://github.com/google/re2.git
-[submodule "third_party/abseil-cpp"]
-	path = third_party/abseil-cpp
-	url = https://github.com/abseil/abseil-cpp.git
 [submodule "third_party/antlr4"]
 	path = third_party/antlr4
 	url = https://github.com/antlr/antlr4.git
```

Comment:
Just curious, but is there anything particular stopping migrating the other libraries as well? Or just out of scope for now?